tests/integrations/python/tests/test_google.py

```
@@ -1678,6 +1678,466 @@ class MathProblem(BaseModel):
         print(f"  Answer: {parsed.final_answer}")
         print(f"  Confidence: {parsed.confidence}")
 
+    @skip_if_no_api_key("gemini")
+    def test_30a_gemini_3_parallel_function_calls_signatures(self, test_config):
```

Ruff (0.14.11)

Unused method argument: `test_config`

(ARG002)